Code/MinimalLib/README.md

```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
 
 ```
 
-## Usage
-
-See the getting started demo at https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html .
-
-Follow the examples of this page to see the various ways to use the JavaScript release of RDKit.
-
 ## Live demos
 
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/demo.html
+If you are using the MinimalLib for the first time, see the getting started examples at https://www.rdkitjs.com/ .
 
-## Contributing
+### All live demos
 
-### Preparing a new release of the package
+- RDKit.js website: https://www.rdkitjs.com/
```

Comment:
This is not live yet, is it?

Reply:
it is!

This is a combination of @greglandrum initial tutorials + the react examples I implemented + css styling + cross reference to related websites (rdkit, npm, rdkit-js source code)

I currently own the domain on Vercel but happy to transfer it to anyone

Reply:
@MichelML Ah, right - I can access it from my personal phone but not from the corporate network. I'll submit a request to whitelist it. Sorry for the false alarm!